core/src/main/java/org/apache/calcite/adapter/enumerable/RexImpTable.java

```diff
@@ -327,6 +328,7 @@
     defineMethod(SOUNDEX, BuiltInMethod.SOUNDEX.method, NullPolicy.STRICT);
     defineMethod(DIFFERENCE, BuiltInMethod.DIFFERENCE.method, NullPolicy.STRICT);
     defineMethod(REVERSE, BuiltInMethod.REVERSE.method, NullPolicy.STRICT);
+    defineMethod(MAP_FILTER, BuiltInMethod.MAP_FILTER.method, NullPolicy.STRICT);
```

Comment:
Dummy function to test lambda functionality, actual implementation could be moved to other PR

Reply:
Done